SimpleGame/src/main/scala-2.11/edu/gemini/experimental/jluhrs/EventStream.scala

```diff
@@ -0,0 +1,26 @@
+package edu.gemini.experimental.jluhrs
+
+/**
+  * Created by jluhrs on 6/2/16.
+  */
+
+// My attempt to describe external inputs as a stream of values with a timestamp.
+
+object EventStream {
+  import Timer._
+
+  type Event[A] = (Time, A)
+  type EventStream[A] = Stream[Option[Event[A]]]
```

Comment:
Why `Option[Event[A]]`?  Is there a difference between a `None` entry and just not having an entry?